scala/scala.bzl

```diff
@@ -200,13 +202,17 @@ def _lib(ctx, non_macro_lib):
 
   if not non_macro_lib:
```

Comment:
I think the defensive thing is to move this above line 198. We don't have a test that fails now, but macros should need scala-reflect at compile time (maybe not runtime). It may be that somehow we are putting scala reflect on every classpath elsewhere.